reference/conanfile/methods.rst

```diff
@@ -1508,6 +1508,14 @@ self.folders
   method in a ``conanfile.py``. In the cache, when running the :command:`conan create`, this subfolder will be relative to the root
   build folder and when running the :command:`conan imports` command it will be relative to the current working directory.
 
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases>`_
+
+- **self.folders.root** (Defaulted to None): Specifies a parent directory where the sources, generators, etc are located specifically when the ``conanfile.py`` is located in a spereated subdirectory.
+
+Available since: `1.51.0 <https://github.com/conan-io/conan/releases>`_
+
+- **self.folders.subproject** (Defaulted to None): Specifies a subfolder where to the ``conanfile.py`` relative  to the project root. This is particulary useful for :ref:`package_layout<example_multiple_subprojects>`.
+
```

Comment:
Perhaps, we could change the above:

```
Read about the feature :ref:`here<package_layout>`.
```
And put it in this line:
```
Read more information about layouts:
* :ref:`Package Layout <package_layout>`.
```

Reply:
```suggestion
* :ref:`Package Layout <package_layout>`.

```

Reply:
I personally like the specific links close to the text, someone might not think to scroll up and find the example?